<launch>

    <!-- 摄像头 -->
    <node name="usb_cam" pkg="usb_cam" type="usb_cam_node" respawn="true" output="screen">
        <param name="video_device" value="/dev/video0" />
        <param name="image_width" value="640" />
        <param name="image_height" value="480" />
        <param name="pixel_format" value="yuyv" />
        <param name="camera_frame_id" value="usb_cam" />
        <param name="io_method" value="mmap" />
    </node>

    <!-- 畸变矫正 -->
    <node ns="usb_cam" name="image_proc" pkg="image_proc" type="image_proc" respawn="true" output="screen" />

    <!-- 显示原图 -->
    <node name="image_view" pkg="image_view" type="image_view" respawn="false" output="screen">
        <remap from="image" to="/usb_cam/image_rect_color" />
        <param name="autosize" value="true" />
    </node>

    <!-- apriltag服务 -->
    <include file="$(find cv_arm_control)/launch/apriltag_server.launch" />

    <node name="testTagDetect" pkg="cv_arm_control" type="testTagDetect.py" respawn="false" output="screen" />
</launch>